ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_virtual_topologies.schema.yml

```
@@ -0,0 +1,121 @@
+            * if the policy is associated with the default VRF, a special control-plane rule is injected
+              in the policy with index `1` referring to a control-plane load-balance policy as defined under
+              `control_plane_virtual_topology`.
+
+
+          For CV Pathfinder, each item in the list creates:
+            * one policy with:
+                * one `match` entry per `application_virtual_topologies` item ordered as in the model.
+                * one last match entry for the `default` application-profile using `default_virtual_topology` information.
+            * one profile per `application_virtual_topologies` item.
+            * one profile for the `default_virtual_topology`..
+            * one load-balance policy per `application_virtual_topologies`.
+            * one load_balance policy for the `default_virtual_topology`.
+            * if the policy is associated with the default VRF, a special control-plane profile is configured
+              and injected first in the policy assigned to the `default` VRF. This profile points to a
+              control-plane load-balance policy as defined under `control_plane_virtual_topology`.
+
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Name of the AVT policy.
+            application_virtual_topologies:
+              type: list
+              description: List of application specific virtual topologies.
+              primary_key: application_profile
+              items:
+                type: dict
+                $ref: "eos_designs#/$defs/virtual_topology"
+                keys:
+                  application_profile:
+                    type: str
+                    description: The application profile to use for this virtual topology. It must be a defined `application_profile`.
+                  name:
+                    type: str
+                    description: Optional name, if not set `<policy_name>_<application_profile>` is used.
+                  id:
+                    type: int
+                    $ref: "eos_cli_config_gen#keys/router_adaptive_virtual_topology/keys/vrfs/items/keys/profiles/items/keys/id"
+                    min: 2
+                    max: 253
+                    description: |-
+                      ID of the AVT in each VRFs. ID must be unique across all virtual topologies in a policy.
+                      ID 1 is reserved for the default_virtual_toplogy.
+                      ID 254 is reserved for the control_plane_virtual_topology.
+            default_virtual_topology:
+              type: dict
+              description: |-
+                Default match for the policy.
+                If defined, ID is hardcoded to 1 for the AVT profile in CV Pathfinder mode.
+                If no default match should be configured, set `drop_unmatched` to `true`.
```

It is required. And since we now have a default name, this description should probably be adjusted. (below suggestion is not great, and I am not sure how this is done for autovpn. So please adjust)
```suggestion
              description: |-
                Default match for the policy.
                If no default match should be configured, set `drop_unmatched` to `true`.
                Otherwise, in CV Pathfinder mode, a default AVT profile will be configured with ID 1.
```

---

[51ff14d](https://github.com/aristanetworks/ansible-avd/pull/3446/commits/51ff14dd7f30b66a8da19268a5c3e3fa71643f95)